packages/go_router/example/lib/shared/data.dart

```diff
@@ -227,42 +227,3 @@ class Repository2 {
     return FamilyPerson(family: family, person: family.person(pid));
   }
 }
-
-/// A state stream.
-abstract class StateStream<T> {
```

Comment:
I'd like to get rid of this shared data file for examples. A TODO or an issue is probably good for now.

Reply:
I am a bit neutral on this. It is useful to put some reusable mock like logininfo in here, but i do agree some of the stuff here probably should just in the example file. what do you think?